doc/level1.rst

```diff
@@ -275,6 +275,24 @@ above rules, such headers will be judged invalid, and discarded by
 level 1 MUAs.  Such an update to the specification will also have
 to describe how an updated MUA will deal with multiple valid headers.
 
+A MUA MUST NOT send an Autocrypt header above 10 KiB in size
+(including the Autocrypt: prefix and folding whitespace).
+Conversely, MUAs SHOULD treat an Autocrypt header
+larger than 10 KiB in size as invalid.
+
+.. note::
+
+  - An Autocrypt header following this specification
+    will be up to 3 KiB in size.
+    Some users might import 4096-bit RSA keys,
+    which may result in an Autocrypt header of up to 5 KiB.
+  - The 10 KiB limit was chosen to accomodate for these use cases
+    but also avoid headers being truncated or emails being rejected
+    by MTAs with a 10 KiB header size limit.
+  - Sending an oversized autocrypt header will result in consistent
+    behaviour accross various clients.
```

Comment:
This is phrased in an oddly positive way now, as if sending an oversized header was intended behavior ("will result in consistent behavior"). I suggest we drop this line and clarify it in the non-note paragraph above:

```
Headers larger than the 10kb are likely the result of unintentionally bloated OpenPGP data. To ensure a consistent failure mode across clients, MUAs SHOULD treat such headers as invalid.
```

Reply:
This goes back to a typical postel discussion: If there is a usable key in the keydata, why disregard it? Wouldn't it better to accept it and not get in the user's way?

I would say: To an extent. If the key is 8kb in size (perhaps due to a third party signature), it's fine, don't bother. If it's 1MB in size (as an unminimized key coming from a user's keyring might well be), it would certainly be better to fail hard enough that users notice. Otherwise it might mask nasty bugs, as happened intially with Enigmail.

Reply:
@Valodim I used a slight variation of the phrase you proposed. I think it makes the reasoning easier to understand.